cmd/vicadmin/vicadm.go

```diff
@@ -140,6 +160,37 @@ func configureReaders() []entryReader {
 	return readers
 }
 
+type authenticator interface {
```

Comment:
This file is a monster already.  Consider moving this interface and the implementation to its own file.